miasm2/core/asmblock.py

```diff
@@ -209,16 +173,16 @@ def get_range(self):
     def get_offsets(self):
         return [x.offset for x in self.lines]
 
-    def add_cst(self, offset, c_t, symbol_pool):
-        if isinstance(offset, (int, long)):
-            l = symbol_pool.getby_offset_create(offset)
-        elif isinstance(offset, str):
-            l = symbol_pool.getby_name_create(offset)
-        elif isinstance(offset, AsmLabel):
-            l = offset
+    def add_cst(self, target, c_t, symbol_pool):
+        if is_int(target):
```

Comment:
It would be great if this API only use `LocKey` argument, instead of accepting several type